sync_sale_picking_cost/models/sale_order_line.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    # The cost whis module is syncing is already being computed in this function.
```

Comment:
Explain this whit the follow nomenclature:

- Precondition: An precondition is something that must be true before a function is invoked (called).
- Postcondition: guarantee that a function makes to its callers, in the form of verifying that one or more conditions are true before it returns
- Return: a statement that determines a value the function will move outside of it, before ending the function execution. By default all functions return None, but that's not very useful.

Example: https://github.com/odoo/odoo/blob/b5a08dcd33cf72c129c4c512045edfe1d8c852a7/addons/account/models/account_account.py#L503-L541

Reply:
@ValentinVinagre I have explained it with a pre and a post. The function does not return anything so i havent redadted the return.